docs/01-Fundamentals-of-Modern-Infrastructures/02-learning-goals.adoc

```diff
@@ -43,13 +48,18 @@ Sie kennen die Leistungsanforderungen an die Infrastruktur und die Limitierungen
 
 // tag::EN[]
 [[LG-1-1]]
-==== LG 1-1: Understand and Distinguish Cloud Types, Cloud Patterns and Cloud Migration Patterns
+==== LG 1-1: Understand and differente between cloud types
```

Comment:
differente => difference?

Reply:
Good catch! I fixed the typo: [dc36d8b](https://github.com/isaqb-org/curriculum-cloudinfra/pull/26/commits/dc36d8b84a70d40645f0bad8e2de4d8736f9ae71)